packages/flutter_markdown/test/flutter_markdown_test.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:test/test.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  test("Simple string", () {
+    testWidgets((WidgetTester tester) {
+      tester.pumpWidget(new MarkdownBody(data: "Hello"));
+
+      Element textElement = tester.findElement((Element element) => element.widget is RichText);
+      RichText textWidget = textElement.widget;
+      TextSpan textSpan = textWidget.text;
+
+      List<Element> elements = _listElements(tester);
+      _expectWidgetTypes(elements, <Type>[MarkdownBody, Column, Container, Padding, RichText]);
+      expect(textSpan.children[0].text, equals("Hello"));
+    });
+  });
+
+  test("Header", () {
+    testWidgets((WidgetTester tester) {
+      tester.pumpWidget(new MarkdownBody(data: "# Header"));
+
+      Element textElement = tester.findElement((Element element) => element.widget is RichText);
+      RichText textWidget = textElement.widget;
+      TextSpan textSpan = textWidget.text;
+
+      List<Element> elements = _listElements(tester);
+      _expectWidgetTypes(elements, <Type>[MarkdownBody, Column, Container, Padding, RichText]);
+      expect(textSpan.children[0].text, equals("Header"));
+    });
+  });
+
+  test("Empty string", () {
+    testWidgets((WidgetTester tester) {
+      tester.pumpWidget(new MarkdownBody(data: ""));
+
+      List<Element> elements = _listElements(tester);
+      _expectWidgetTypes(elements, <Type>[MarkdownBody, Column]);
+    });
+  });
+
+  test("Ordered list", () {
+    testWidgets((WidgetTester tester) {
+      tester.pumpWidget(new MarkdownBody(data: "1. Item 1\n1. Item 2\n2. Item 3"));
+
+      List<Element> elements = _listElements(tester);
+      _expectTextStrings(elements, <String>["1.", "2.", "3."]);
```

Comment:
The test is correct (the item texts are stored as RichText and not compared), but I changed it to look through all text.